clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -335,8 +335,13 @@ public int memberEpoch() {
         return memberEpoch;
     }
 
+    /**
+     * @return True if there hasn't been a call to consumer.poll() withing the max.poll.interval.
+     * In that case, it is expected that the member will leave the group and rejoin on the next
+     * call to consumer.poll().
+     */
     @Override
-    public boolean isStaled() {
+    public boolean isStale() {
```

Comment:
Thank you!!!!! 😄

Reply:
Any reason we don't just expose the inner state via a `state()` method so that we don't have to write `isStateA`, `isStateB`, `isStateC`,  etc.?

Reply:
I would guess that @philipnee preferred the convenience method when he added it, to use it from the HBManager. That being said, your comment made me notice that the isStale is not used anymore there, so removing it